### PR TITLE
LPS-148590 Hide object relationship entries when table is not active

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-148590

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please fix the source formatter issues and resend? Thanks

**Comment by gabrielwas** [comment]
Just started reviewing :)

**Comment by danvb112** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/list/renderer/ObjectEntryTableInfoListRenderer.java [hunk 1]
```diff
@@ -45,10 +51,14 @@
 
 	public ObjectEntryTableInfoListRenderer(
 		InfoItemRendererTracker infoItemRendererTracker,
-		ObjectFieldLocalService objectFieldLocalService) {
```
Comment (marcelabc) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/list/renderer/ObjectEntryTableInfoListRenderer.java:48]: Hey @danvb112, could you sort the parameters please? Thanks!

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hey @danvb112, how are you doing?
I finished your review! Let me know if there is any question :smile:

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
Hey @marcelabc, could you review this PR again? if there is something wrong please tell me :smile:

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by marcelabc** [comment]
hey @danvb112, I'm resending in https://github.com/liferay-objects/liferay-portal/pull/527

**Pull request closed without merging**